components/script/dom/servoparser/xml.rs

```diff
@@ -165,13 +183,7 @@ impl<'a> TreeSink for Sink {
     }
 
     fn complete_script(&mut self, node: Self::Handle) -> NextParserState {
-        let script = node.downcast::<HTMLScriptElement>();
-        if let Some(script) = script {
-            return match script.prepare() {
-                true => NextParserState::Continue,
-                false => NextParserState::Suspend,
-            };
-        }
-        NextParserState::Continue
+        self.script.set(Some(node.downcast().unwrap()));
```

Comment:
This is the only way I found to make xml5ever just stop tokenizing on `</script>` to handle the script itself out of it.